### ISSUE TITLE
Error while building docs related to numpy

### ISSUE THREAD

**Issue by sergiovalmac** [stochasticnetworkcontrol/snc#10]
@IanRDavies, if I go to `cd docs`, `pip install -r requirements.txt`, and `make html`, I get the following error:
```
RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd
```
This seems to be related to upgrading and downgrading `numpy` during installation.

**Comment by IanRDavies** [comment]
I think this should have been fixed in #9. Have you been able to reproduce the issue since that PR merged?

**Comment by sergiovalmac** [comment]
You are right. It is fixed with your PR! Thanks!
